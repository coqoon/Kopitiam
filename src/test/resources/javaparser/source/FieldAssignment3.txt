class Foo {
  int b;
  int a () {
    return 10;
  }
  void foo () {
    b = a();
  }
}
