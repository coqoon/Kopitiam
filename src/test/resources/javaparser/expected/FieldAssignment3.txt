class Foo {
  int b;
  int a () {
    return 10;
  }
  void foo () {
    int tmp_1 = this.a();
    this.b = tmp_1;
  }
}
